src/dataHandler.py

```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
         return
     
     def send_all_data(self, sendingTo):
+        if(sendingTo == blackbox):
```

Comment:
```suggestion
        if sendingTo == 'blackbox':
```
python doesnt need `()` in if statements. I know its weird.